Source/SwiftLintBuiltInRules/Rules/Lint/PrivateSwiftUIStatePropertyRule.swift

```diff
@@ -243,4 +284,18 @@ private extension AttributeListSyntax? {
             return identifier.name.text == "State"
```

Comment:
Suggest to merge both computed properties. Then this is the only line to change.